source/module_hamilt/hamiltlcao.h

```diff
@@ -0,0 +1,106 @@
+#ifndef HAMILTLCAO_H
```

Comment:
Do we need an underscore for file name: `hamilt_lcao`? Would it be better?

Reply:
I don't know, maybe underscore is better? Which format is more common practice?